### DISCUSSION TITLE
Can't figure out how to integrate this into my Sapper project that still uses postcss plugins like postcss-nested

### DISCUSSION THREAD

**Discussion by Evertt** [windicss/windicss#53] [Q&A] [answered]
Hey I'm trying to migrate [my Sapper project](https://github.com/Evertt/sapper-cms/tree/cms) to use windicss right now and I just realized that I still use postcss for plugins like `postcss-import` and `postcss-nested`. So now I'm trying to make my project use both and I can't easily figure out how.

In my `rollup.config.js` I have the following code:

    const { createPreprocessors } = require("./svelte.config.js")

And then in `svelte.config.js` I have this:

    const sveltePreprocess = require("svelte-preprocess")
    const postcss = require("./postcss.config")
    
    const createPreprocessors = ({ sourceMap }) => [
      sveltePreprocess({
        sourceMap,
        defaults: {
          script: "typescript",
          style: "postcss",
        },
        postcss,
      }),
      // You could have more preprocessors, like mdsvex
    ]
    
    module.exports = {
      createPreprocessors,
      // Options for `svelte-check` and the VS Code extension
      preprocess: createPreprocessors({ sourceMap: true }),
    }

Now I'm not sure how to migrate this such that it uses *both* postcss *and* windicss. I tried a few ways of including windicss in that `createPreprocessors` function, but it either results in errors during build or it results in my styling just not being compiled at all... Do you have any advice?

Here's my repo in case you want to see the whole code:

[https://github.com/Evertt/sapper-cms/tree/cms](https://github.com/Evertt/sapper-cms/tree/cms)

**Comment by alexanderniebuhr** [comment]
a good help. Also, sapper not supported fully

https://github.com/windicss/svelte-windicss-preprocess/tree/main/example/outdated/sapper-rollup

**Comment by voorjaar** [comment, marked as answer]
https://github.com/windicss/windicss/issues/114
